kiwi/filesystem/isofs.py

```diff
@@ -61,3 +67,19 @@ def create_on_file(self, filename, label=None, exclude=None):
         iso.relocate_boot_catalog(filename)
         iso.fix_boot_catalog(filename)
         return hybrid_offset
+
+    def _find_iso_creation_tool(self):
+        """
+        There are tools by J.Schilling and tools from the community
+        Depending on what is installed a decision needs to be made
+        """
+        iso_creation_tools = [
+            '/usr/bin/mkisofs', '/usr/bin/genisoimage'
+        ]
+        for tool in iso_creation_tools:
+            if os.path.exists(tool):
+                return tool
+        raise KiwiIsoToolError(
+            'No iso creation tool found, searched for: %s' %
+            iso_creation_tools
+        )
```

Comment:
yes actually it would be against FHS if those tools are not in /usr/bin, but I agree a path search is the way to go. I'm going to change it